loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -1844,11 +1844,16 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 			$(childContainer).hide();
 
-			if (builder.wizard)
+			if (builder.wizard) {
+				L.DomUtil.remove($(container).find('td')[3]);
```

Comment:
use `$(container).find(".loleaflet-annotation-menubar")[0]` instead

Reply:
I got it and thanks, I will take care of it in future